content/case_histories/bookpurnong/survey.rst

```diff
@@ -3,3 +3,67 @@
+
+Frequency domain system
+-----------------------
+
+If the current waveform in the transmitter loop is sinusoidal at a certain frequency, this system is said to operate in frequency domain. For this case, the received waveform is also sinusoidal at the same frequency, but likely with altered magnitude and a phase shift. It is convenient to decompose the receiver waveform to two orthogonal sinusoidal components: one entirely in-phase (real) and the other entirely out-of-phase (imaginary or quadrature). A loop-loop system in frequency domain can be understood with a demonstrative 3-loop model, please read the tutorial (link). You may find the following concepts useful. Click the links to learn more in Geophysical Surveys.
+
+* Skin depth
+* Footprint of a system
+* Separation of source and receiver
+* Resistive and conductive limits
+* Scaling with induction number
+* Height above the surface
+* Loop coupling
+* Data representation
+
+.. figure:: ./images/booky-resolve.jpg
+    :align: left
+    :scale: 80% 
+    :name: booky-resolve
+    
+The figure above shows the frequency-domain loop-loop system used at Bookpurnong. The commercial name of the system is RESOLVE, operated by Fugro Airborne (now CGG). It operates five pairs of horizontal co-planar (HCP) loop-loop configurations at 382, 1822, 7970, 35920 and 130100 Hz, and one pair of vertical co-axial loop-loop configuration at 3258 Hz. The measured secondary magnetic field (Hs) is represented as the ratio to the primary field (Hp) in part per million.
+
+Time domain system
+------------------
+
+If the transmitter current waveform is not sinusoid, it is said to operate in time domain. Usually the time-domain transmitter waveform is designed as an abrupt pulse or step off (on-time) followed by a no-current quiet peroid (off-time). The turn-off of the primary field excites the secondary current and field in the ground. The receiver loop measures the time-decaying transient at a number of time channels during the off-time when the primary field is absent. In theory, a time-domain system is equivelant to the convolution of the transmitter waveform with the inverse Fourier transform of a multi-frequency frequency-domain system. Early time channels are roughly equal to higher frequencies. Unlike a frequency-domain system, the time-domain data re real values of the actual field. Direct measurement of the magnetic field (B in Tesla) is possible, but people usually measure the time derivatives of the magnetic field (dB/dt in volt) for logistic reasons. You may find the following concepts useful. Click the links to learn more in Geophysical Surveys.
```

Comment:
"In theory, a time-domain system is equivelant to the convolution of the transmitter waveform with the inverse Fourier transform of a multi-frequency frequency-domain system. " : the formulation seemed confusing for me.